documentation/docs/components/timer.md

```diff
@@ -0,0 +1,29 @@
+# Timer
+## Example
+
+| type      | name         |parameter_list             |
+| --------- | ------------ |---------                  |
+|timer	    |timer_par_5_1 |starting_minutes: 10       |
+|timer      |timer_par_5_2 |starting_minutes: 15       |
+
+![](images/timer.png)
+
+[Google Sheet Demo](https://docs.google.com/spreadsheets/d/1jQAvELG5EXj6zgj0bBQmVg7Iaa-66OpemdZzgOh2PP0/edit#gid=569531329)   
+[Live Preview Demo](https://plh-global.web.app/template/comp_timer)
+
+## Parameters
+
+| Parameter             | Value           | Description                 |
+| ---------             | -----------     |  ---------                  |                            
+|title                  |null (default)   | No title                    |
+|title                  |string           | Any text as a title         |                  
+|help                   |null (default)   | No help tooltip             |
+|help                   |string           | Any text for help tooltip   |                  
+|is_editable_on_playing |false (default)  | Not possible to change the values of the timer when it is running|
+|is_editable_on_playing |true             | Possible to change the values of the timer when it is running| 
+|duration_extension     |1 (default)      | Value for increasing the time on the timer|                        
```

Comment:
Is this the value in minutes?

Reply:
1. As indicated on the template parameter list, the value is in minutes. 

![image](https://user-images.githubusercontent.com/89448221/173790355-5692794f-9dd7-4724-acf0-abf2933b318a.png)

2. I may need further clarification for the last parameter name which is set to seconds.

![image](https://user-images.githubusercontent.com/89448221/173790604-8d0016b1-20d7-41f2-8b02-00e3aa35fa21.png)